### PR TITLE
fix(PE-7259): make ticker dynamic

### PR DESCRIPTION
**Pull request by atticusofsparta**

### CODE REVIEW

File: src/state/contexts/GlobalState.tsx [hunk 1]
```diff
@@ -89,15 +89,15 @@ export function GlobalStateProvider({
   const [updatingTicker, setUpdatingTicker] = useState(false);
 
   useEffect(() => {
-    if (state.ioTicker === initialState.ioTicker && !updatingTicker) {
+    if (!updatingTicker) {
```
Comment (kunstmusik) [inline src/state/contexts/GlobalState.tsx:92]: Not sure why there is updatingTicker at all here? What is it protecting against?

File: src/state/contexts/GlobalState.tsx [hunk 2]
```diff
@@ -89,15 +89,15 @@ export function GlobalStateProvider({
...
   }, []);
 
   async function updateTicker() {
     try {
       setUpdatingTicker(true);
-      const ticker = 'tIO'; // TODO, use contract to get ticker
+      const ticker = (await state.arioContract.getInfo()).Ticker;
       dispatchGlobalState({ type: 'setIoTicker', payload: ticker });
     } catch (error) {
       console.error(error);
```
Comment (kunstmusik) [inline src/state/contexts/GlobalState.tsx:103]: might be worth adding a backup here to set to "ARIO"

Comment (atticusofsparta) [reply src/state/contexts/GlobalState.tsx]: [040afc3](https://github.com/ar-io/arns-react/pull/615/commits/040afc3e6a79e21822c6ca030ab167a64001f4b4)

File: src/state/contexts/GlobalState.tsx (hunk 1 above)
Comment (atticusofsparta) [reply src/state/contexts/GlobalState.tsx]: [040afc3](https://github.com/ar-io/arns-react/pull/615/commits/040afc3e6a79e21822c6ca030ab167a64001f4b4)

**Review by kunstmusik — APPROVED** [review]

**Pull request merged**